tests/functions/test_transfer.py

```diff
@@ -90,6 +106,25 @@ def test_execute(func, variable, params, fail, expected, benchmark, mode):
     assert np.allclose(res, expected)
     benchmark(f.function, variable)
 
+@pytest.mark.function
+@pytest.mark.transfer_function
+@pytest.mark.transfer_function
```

Comment:
transfer_function mark is added twice.

it'd be nice to mark as benchmark and assign benchmark group.